test/e2e/specs/change-detection.test.js

```diff
@@ -52,6 +57,20 @@ describe( 'Change detection', () => {
 		await page.setRequestInterception( false );
 	}
 
+	it( 'Should autosave post', async () => {
+		await page.type( '.editor-post-title__input', 'Hello World' );
+
+		const isSavingState = await page.waitForSelector( '.editor-post-saved-state.is-saving' );
+		const autosaveState = await ( await isSavingState.getProperty( 'innerText' ) ).jsonValue();
+
+		expect( autosaveState ).toBe( __( 'Autosaving' ) );
```

Comment:
Related to https://github.com/WordPress/gutenberg/pull/6257#issuecomment-388903016 , the `__` is misleading as it's not doing anything for us. The evaluated value in the context of this file is based on the instance of `@wordpress/i18n` running in the Node process, whereas the value in the browser is based on the copy running in the browser window. If, theoretically, we were running a non-English locale in our test site, this would fail if that non-English locale data were not also set in the Node process. In other words, this appears as though we're comparing the two localized strings, but it's not providing any more benefit than just:

```js
expect( autosaveState ).toBe( 'Autosaving' );
```

Reply:
i'll remove the translation